AniHyou/Screens/StaffDetails/Views/StaffDetailsView.swift

```diff
@@ -110,7 +110,15 @@ struct StaffDetailsView: View {
     @ViewBuilder
     var staffMedia: some View {
         LazyVStack(alignment: .leading) {
```

Comment:
for fixing the picker width, try to change `LazyVStack` to a `List` and apply the `.listStyle(.plain)` to it.